### PR TITLE
support set oss bucket stroage class

### PR DESCRIPTION
**Pull request by huiguangjun**

### CODE REVIEW

File: alicloud/resource_alicloud_oss_bucket.go [hunk 1]
```diff
@@ -204,8 +204,10 @@ func resourceAlicloudOssBucket() *schema.Resource {
 				Computed: true,
 			},
 			"storage_class": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:         schema.TypeString,
+				Default:      oss.StorageStandard,
+				Optional:     true,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_oss_bucket.go]: Can the storage_class be modified? If not, please set `ForceNew` to the parameter.

File: alicloud/validators.go [hunk 2]
```diff
@@ -754,6 +754,18 @@ func validateOssBucketObjectServerSideEncryption(v interface{}, k string) (ws []
...
+			errors = append(errors, fmt.Errorf(
+				"%q must be a valid  StorageClass value , expected %s, %s or %s, got %q",
+				k, oss.StorageStandard, oss.StorageIA, oss.StorageArchive, strorage))
+		}
+	}
+	return
+}
+
 func validateDomainName(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
```
Comment (xiaozhu36) [inline alicloud/validators.go]: Please move the parameter `storage_class ` to `Argument Reference` in the oss bucket docs.

File: alicloud/resource_alicloud_oss_bucket.go (hunk 1 above)
Comment (huiguangjun) [reply alicloud/resource_alicloud_oss_bucket.go]: add the ForceNew to the storage_class parameter

File: alicloud/validators.go (hunk 2 above)
Comment (huiguangjun) [reply alicloud/validators.go]: done.

**Pull request merged**